src/site/content/en/handbook/effective-instruction/index.md

```diff
@@ -5,22 +5,22 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  Strategies for writing effective instruction on web.dev, especially for adult learners.
+  Learn strategies for writing effective instruction on web.dev, especially for adult learners.
 ---
 
 Most web.dev content includes at least some instruction. These guidelines help make your instruction easy to follow.
 
-Explain _why_ to do something before explaining _how_ to do it. Knowing what they're trying to accomplish helps readers understand the purpose of individual steps.
+Explain _why_ to do something before explaining _how_ to do it. Knowing the end goal helps readers understand the purpose of individual steps.
 
 Keep the scope of instruction manageable. If a task is complex, find ways to chunk the steps. For example, provide the essential pieces in a post and put the ancillary pieces in one or more codelabs.
 
-Always provide examples for complex or abstract concepts. If you're debating whether to give an example, do.
+Always provide examples for complex or abstract concepts. If you're debating whether to give an example, add one.
 
-Use web.dev's built-in [Instruction components](/handbook/web-dev-components#instruction) for instructing common tasks in the tools often referenced on the site (e.g., Glitch, Chrome DevTools). This approach provides a consistent user experience and makes it easy to update instructions across the site when needed.
+Use web.dev's built-in [Instruction components](/handbook/web-dev-components#instruction) for common tasks that are often referenced on the site (for example, instructions about opening Glitch or Chrome DevTools). This approach provides a consistent user experience and makes it easy to update instructions across the site when needed.
```

Comment:
I like that we explain the why here (it follows the handbook principles to explain things). Should we do this more often when we list style rules? Or is it clear enough why something is a rule?

Reply:
This is something I was struggling with--balancing robust explanation and skimmability to support different purposes for reading.

If we had a progressive disclosure control whose salience worked a bit better for this context than the `detail` widget we have now, I'd definitely provide more explanations. (Basically, I want something like the [Material Design Writing](https://material.io/design/communication/writing.html#principles) page.)

I could perhaps work on that with @robdodson if you're willing to flag cases where you think additional explanation or rationales would be helpful.

Reply:
Also, mostly really like the edit, which solves some phrasing problems I was having trouble with, but would like to keep a reference to tools if possible since that really is the limiting factor--we have instructions for Glitch and DevTools because those are the tools referenced most often in posts.